Sources/Operators.swift

```diff
@@ -241,7 +241,7 @@ public func <~~ (key: String, json: JSON) -> [URL]? {
 /**
 Encode custom operator.
 */
-infix operator ~~> { associativity left precedence 150 }
+infix operator ~~> : MultiplicationPrecedence
```

Comment:
i agree it's more expressive. would you mind adding that custom precedence with this PR?